rpcd/playbooks/roles/rpc_maas/tasks/maas_exclude.yml

```diff
@@ -13,18 +13,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Ensure local checks are removed
-  file:
-    path: "/etc/rackspace-monitoring-agent.conf.d/{{ item }}--{{ inventory_hostname }}.yaml"
-    state: absent
-  with_items:
-    - "{{ maas_excluded_checks }}"
-  delegate_to: "{{ physical_host }}"
```

Comment:
This seems redundant with the second task below.  The task below looks through all of the files in `/etc/rackspace-monitoring-agent.conf.d/` to find any which match the regexes in `maas_excluded_checks`.